File: Ejercicios Corredor/Trabajo Corredor/Texto.cs
public class Texto {
    public string Nombre = "Kevin";
    public string Apellido = "Andrey";
    public string Ciudad = "Bogotá";
    public string Pais = "Colombia";
    public string Profesion = "Desarrollador";

    public string Concatenar(string a, string b) => $"{a} {b}";
    public int Contar(string texto) => texto.Length;
    public string Mayuscula(string texto) => texto.ToUpper();
    public string Minuscula(string texto) => texto.ToLower();
    public bool Contiene(string texto, string palabra) => texto.Contains(palabra);
}

File: Ejercicios Corredor/Trabajo Corredor/Texto.go
package main
import (
	"fmt"
	"strings"
)

type Texto struct {
	nombre    string
	apellido  string
	ciudad    string
	pais      string
	profesion string
}

func (t Texto) Concatenar(a, b string) string { return a + " " + b }
func (t Texto) Contar(texto string) int { return len(texto) }
func (t Texto) Mayuscula(texto string) string { return strings.ToUpper(texto) }
func (t Texto) Minuscula(texto string) string { return strings.ToLower(texto) }
func (t Texto) Contiene(texto, palabra string) bool { return strings.Contains(texto, palabra) }

func main() {
	fmt.Println(Texto{}.Concatenar("Hola", "Mundo"))
}
